.github/workflows/pre-merge.yaml

```diff
@@ -10,11 +10,22 @@ on:
 
 jobs:
   gradle:
-    runs-on: ubuntu-latest
+    runs-on: [ubuntu-latest]
+    strategy:
+      fail-fast: false
```

Comment:
This line meant to not fail build if we have issues on 1.4? 
Since we shouldn't have issues on 1.3.72 obviously.

If so, maybe change it to `true` or remove? To enforce preparations for 1.4.

Reply:
> This line meant to not fail build if we have issues on 1.4?

No it means that if one of the two builds fail, the other will be aborted.
I generally prefer to see the result of both builds to easily understand if the issue is on the Kotlin preview or affecting both builds.